### ISSUE TITLE
dhtnode: SIGINT fails when trying to bind on low port numbers

### ISSUE THREAD

**Issue by ghenry** [savoirfairelinux/opendht#577]
Using:

db000a2f8849584f84a26cba8583a188741e0418 (HEAD -> master, tag: 2.3.2, origin/master, origin/HEAD)

```
./dhtnode -b bootstrap.sentrypeer.org -p 422 -v
[184037.701954] Can't bind inet socket: DhtException occurred: Can't bind socket on 0.0.0.0:422 Permission denied
[184037.702047] Can't bind inet6 socket: DhtException occurred: Can't bind socket on [::]:422 Permission denied

DhtException occurred: Can't bind socket
^C^[[A^C^C
quit
^C^C^C^C^Z^C^C^C^C^C^C^C^C^C^C^C
```

**Comment by ghenry** [comment]
Ctrl-Z too fails.

**Comment by ghenry** [comment]
Had to `kill -9` the process.

**Comment by aberaud** [comment]
Fixed in 2.3.5
